src/test/java/teammates/test/cases/ui/browsertests/InstructorFeedbackEditPageUiTest.java

```diff
@@ -160,7 +160,39 @@ private void testEditSessionAction() throws Exception {
         feedbackEditPage.reloadPage();
         feedbackEditPage.verifyHtmlMainContent("/instructorFeedbackEditSuccess.html");
 
-
+        ______TS("test two 'change' links' functionality to expand uncommon settings panels");
+        
+        // test uncommon settings for 'send emails'
+        feedbackEditPage.clickEditUncommonSettingsSendEmailsButton();
+        feedbackEditPage.verifyEditSessionBoxIsEnabled();
+        feedbackEditPage.toggleClosingSessionEmailReminderCheckbox();
+        feedbackEditPage.clickSaveSessionButton();
+        feedbackEditPage.reloadPage();
```

Comment:
@wkurniawan07 What we discuss yesterday was to use`BackDoor.getFeedbackSession(..)` to verify whether the `closingSessionEmailReminder` settings has been changed successfully. 

I don't get the point and I think this has nothing to do with 'wait' for the datastore to be persistent. The `BackDoor` actually do POST request to the server (which is essentially the same as you reload the page (GET request)) and get data by JSON file. If the data still don't gets enough time to be persistent. The data gotten by `BackDoor` will still be the old one (wrong one)(but this usually won't happen as the operations such as forming parameters, sending POST request and etc. actually takes longer time than persist the data storage).

I think the data persistence part is specified by App Engine, which is not sth we can change. Thus, what we can do is to add some statements (which is meaningful and it is not dummy) between them to give enough time for data storage.